evm/v0.5/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "incremental": true,
+    "target": "es2019",
+    "module": "commonjs",
+    "allowJs": true,
+    // "sourceMap": true,
```

Comment:
Are you planning on turning these back on? Otherwise I think we should delete commented out code.

Reply:
Good catch, ill clean this up